### PR TITLE
[function] SELECT VERSION()

### LINKED ISSUE #762: [function] SELECT VERSION()
**Summary**

Return the current version information(ctx.version) of FuseQuery.
This is a context function as:
https://github.com/datafuselabs/datafuse/blob/master/fusequery/query/src/functions/context_function.rs

### PR DESCRIPTION
**Pull request by PsiACE**
Signed-off-by: Chojan Shang <psiace@outlook.com>

I hereby agree to the terms of the CLA available at: https://datafuse.rs/policies/cla/

## Summary

try add version() function.

```
mysql> SELECT VERSION();
+-----------------------+
| VERSION(FuseQuery v-) |
+-----------------------+
| FuseQuery v-          |
+-----------------------+
1 row in set (0.00 sec)
```

## Changelog

- New Feature

## Related Issues

Fixes #762

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by PsiACE** [comment]
Basically finished, just need some tests and documentation. 

@BohuTANG Could you please review it? 

cc @drmingdrmer , I checked the output file, oops, vargen can't gen `FUSE_COMMIT_VERSION`. Just get `cargo:rustc-env=FUSE_COMMIT_VERSION=`.

File: common/functions/src/udfs/version.rs [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// Copyright 2020-2021 The Datafuse Authors.
```
Comment (BohuTANG) [inline common/functions/src/udfs/version.rs:1]: Cool, but why we need the udfs function?

Comment (BohuTANG) [reply common/functions/src/udfs/version.rs:1]: Sorry, got it.
It's ok to me.

**Comment by drmingdrmer** [comment]
> cc @drmingdrmer , I checked the output file, oops, vargen can't gen `FUSE_COMMIT_VERSION`. Just get `cargo:rustc-env=FUSE_COMMIT_VERSION=`.

Sorry but I do not get it. Have you had a problem using this const var? What do you expected and what did you actually get?

The `build.rs` generates `FUSE_COMMIT_VERSION` as an env var. `Config` loads the env and saves it in a const var.

https://github.com/datafuselabs/datafuse/blob/c2cc5952dd9fd7e6adad19389db8bfdf654a4c0c/common/building/src/lib.rs#L49-L61

https://github.com/datafuselabs/datafuse/blob/c2cc5952dd9fd7e6adad19389db8bfdf654a4c0c/fusequery/query/src/configs/config.rs#L10

**Comment by PsiACE** [comment]
> What do you expect and what did you actually get?

I may need a value, but get nothing.

**Comment by BohuTANG** [comment]
> > What do you expect and what did you actually get?
> 
> I may need a value, but get nothing.

Here we should use:
config::FUSE_COMMIT_VERSION
https://github.com/datafuselabs/datafuse/blob/master/fusequery/query/src/bin/fuse-query.rs#L33-L36

**Comment by drmingdrmer** [comment]
> > What do you expect and what did you actually get?
> 
> I may need a value, but get nothing.

Confirmed It's my fault. Working on it.

**Comment by sundy-li** [comment]
Is it possible to register the feature simd to the version description?

Currently, the arrow is not using simd feature, even we have 
```
[features]
simd = ["arrow/simd"]
```

Query: 

```
mysql> SELECT sum(number % 3) FROM numbers_mt(10000000000);
+-------------------+
| sum((number % 3)) |
+-------------------+
|        9999999999 |
+-------------------+
1 row in set (12.58 sec)
```

Profile flamegraph:

image


Need to investigate that.

**Comment by PsiACE** [comment]
> Is it possible to register the feature simd to the version description?

yep, i guess we can detect this feature.

> Currently, the arrow is not using simd feature, even we have
> 
> ```
> [features]
> simd = ["arrow/simd"]
> ```

now, this feature is not enabled by default, try `--features simd`.

**Comment by BohuTANG** [comment]
Show SIMD feature in the version, we want it.

**Comment by PsiACE** [comment]
Now, we can get version info like:

```
+----------------------------------------------------------------------------------------+
| VERSION(FuseQuery v-0.1.0-623cfb1(1.54.0-nightly-2021-06-09T07:40:55.026718153+00:00)) |
+----------------------------------------------------------------------------------------+
| FuseQuery v-0.1.0-623cfb1(1.54.0-nightly-2021-06-09T07:40:55.026718153+00:00)          |
+----------------------------------------------------------------------------------------+
```

Or simd version like:

```
+---------------------------------------------------------------------------------------------+
| VERSION(FuseQuery v-0.1.0-623cfb1-simd(1.54.0-nightly-2021-06-09T07:46:11.570439919+00:00)) |
+---------------------------------------------------------------------------------------------+
| FuseQuery v-0.1.0-623cfb1-simd(1.54.0-nightly-2021-06-09T07:46:11.570439919+00:00)          |
+---------------------------------------------------------------------------------------------+
```

thanks @drmingdrmer 

cc @BohuTANG

**Comment by drmingdrmer** [comment]
@PsiACE nice shot man!

File: website/datafuse/docs/sqlstatement/information-functions/database.md [hunk 2]
```diff
@@ -8,16 +8,16 @@ Returns the name of the currently selected database. If no database is selected,
...
 ## Examples
 
 ```
 mysql> SELECT DATABASE();
-+------------+
-| database() |
-+------------+
-| default    |
-+------------+
++-------------------+
```
Comment (BohuTANG) [inline website/datafuse/docs/sqlstatement/information-functions/database.md:18]: :100:

File: website/datafuse/mkdocs.yml [hunk 3]
```diff
@@ -152,6 +152,7 @@ nav:
           - SIPHASH: sqlstatement/hash-functions/siphash.md
       - Information Functions:
           - DATABASE: sqlstatement/information-functions/database.md
+          - VERSION: sqlstatement/information-functions/version.md
```
Comment (BohuTANG) [inline website/datafuse/mkdocs.yml:155]: +1

**Review by BohuTANG — APPROVED** [review]
LGTM

**Pull request merged**